app/uk/gov/hmrc/gform/gform/DeclarationController.scala

```diff
@@ -197,11 +197,15 @@ class DeclarationController(
           cache.formTemplate,
           data,
           cache.form.envelopeId)
-        _ <- gformConnector.submitFormWithPdf(
-              FormId(cache.retrievals, formTemplateId, maybeAccessCode),
-              customerId,
-              htmlForPDF,
-              AffinityGroupUtil.fromRetrievals(cache.retrievals))
+        _ <- new GformSubmission {}
```

Comment:
Cannot we use `object GformSubmission extends GformSubmission`, and use static reference: `GformSubmission. handleSubmission`?

Reply:
I don't generally like static obj but in this case it wont harm so....on the way